terraform/envs/integration/main.tf

```diff
@@ -0,0 +1,19 @@
+locals {
+  labels = {
+    project     = var.project
+    tribe       = "test"
+    squad       = "QaNightlyresults"
+    environment = var.environment
+    github_hash = var.hash_id
+  }
+}
+
+module "apps" {
+  source                    = "../modules/apps"
+  labels                    = local.labels
+  app_version               = var.app_version
+  project                   = var.project
+  image                     = "gcr.io/prestashop-cloud-integration/testing-qanightly-migration/qanightlyresults"
+  service_account_email     = "builder-qanighlty@prestashop-cloud-integration.iam.gserviceaccount.com"
```

Comment:
I used this account temporary, but I supposed the project need a different SA to have access (I didn't see the information in kubernetes files)